README.md

```diff
@@ -91,60 +110,32 @@ This project also uses ESLint and sass-lint to help find bugs and enforce code q
 Run `yarn license-report` to generate a license report for all npm packages. See output in `license-report.txt`.
 
 
-## 2. CICD Deploy
+## DUS Modes:
 
-### Requirements
+### Classic Mode
+This is first release of the DUS solution. The major services included in this mode include Amazon Elasticsearch, Amazon Textract, Amazon Comprehend and Amazon Comprehend Medical that allow digitization, information extraction and indexing in DUS.
 
-- aws cli
+### Kendra-Enabled Mode
 
-### Getting Started with Full Deploy
+In the Classic version, DUS supports searching/indexing of documents using Amazon Elasticsearch
```

Comment:
Should we add information for cost due to kendra index?

Reply:
Updated in the Cost section